PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskJetExtractor.cxx

```diff
@@ -513,13 +513,14 @@ AliAnalysisTaskJetExtractor::AliAnalysisTaskJetExtractor() :
   fTruthMaxLabel(100000),
   fHadronMatchingRadius(0.4),
   fJetMatchingRadius(0.3),
+  fSharedPtFraction(0.5),
   fMatchedDetLevelJetsArrayName(""),
```

Comment:
Do we still need all these variables (fMatched*LevelJets*)?

Reply:
No we shouldn't. I left them in order to see what we were going to do about the point below with InitializeTree.

Reply:
Resolved.